### DISCUSSION TITLE
댓글 삭제와 관련하여

### DISCUSSION THREAD

**Discussion by cho-ja-young** [MajorIn-ABO/MajorIn_FE#23] [Ideas]
좋아요 같은 경우에는 아예 기록 자체를 삭제하는 것으로 정했는데, 

댓글은 유저가 삭제했을 때, 백에서는 삭제 날짜만 저장시키고 프런트단에서 그냥 "삭제된 댓글 입니다" 이런식으로 보여지게(? 설정하는거 어때?
혹시나 이상한 댓글쓰고 튀는 사람들 신고해야하면,,, 그 데이터가 있어야할 수도 있지 않나 싶어서

**Comment by cho-ja-young** [comment]
아 생각해보니까 일단 삭제는,,,, 내가 쓴 댓글인지 확인 후에 가능하네... 이것부터 일단 코드 추가해야겠다.

**Reply by wlsl10** [reply]
그렇겠다. 아니면 신고된 댓글이 아니면 남겨둘 필요가 없으니까 신고가 된 경우에만 삭제를 못하게 하거나 삭제해도 데이터가 남아있게 따로 설정하는 건 어때?
